bpfman/src/lib.rs

```diff
@@ -61,11 +61,11 @@ const MAPS_USED_BY_PREFIX: &str = "map_used_by_";
 
 pub(crate) mod directories {
     // When release is made, the dispatcher images should be changed to the
-    // release tag, for example "v0.5.4". After release is made, the tag should
+    // release tag, for example "v0.5.5". After release is made, the tag should
     // be changed back to the "latest" release tag. Note also that these values should
     // be read through the public interfaces provided by config::Config's RegistryConfig
-    pub(crate) const XDP_DISPATCHER_IMAGE: &str = "quay.io/bpfman/xdp-dispatcher:latest";
-    pub(crate) const TC_DISPATCHER_IMAGE: &str = "quay.io/bpfman/tc-dispatcher:latest";
+    pub(crate) const XDP_DISPATCHER_IMAGE: &str = "quay.io/bpfman/xdp-dispatcher@sha256:61c34aa2df86d3069aa3c53569134466203c6227c5333f2e45c906cd02e72920";
```

Comment:
Since we're going with the sha tag option, you should delete the comments above about changing them and changing them back.  Also, make a similar change to RELEASE.md.

Reply:
I updated the comment. I have a separate PR with some nits to RELEASE.md and already removed that step.